installer/mac/update.sh

```diff
@@ -38,37 +38,37 @@ function printError(){
 function verifySetup(){
     if [ "$APP_DIR" == "$EMPTY_STRING" ]
     then
-        printError "Brackets app dir is missing"
+        printError "Brackets app dir is missing: INS_O1"
```

Comment:
Don't we need "Error" word in these strings?

Reply:
"ERROR:" String will be added by printError Function so we don't need to add explicitly.